src/main/java/com/mappy/fpm/batches/tomtom/dbf/lanes/LaneTagger.java

```diff
@@ -9,7 +9,6 @@
 import static com.google.common.base.Joiner.on;
 import static com.google.common.collect.Lists.reverse;
 import static com.google.common.collect.Maps.newHashMap;
-import static com.mappy.fpm.batches.tomtom.helpers.RoadTagger.isReversed;
```

Comment:
We now don't rely on the ONEWAY field, but compute the direction using the restrictions tables.

The result is now computed, and the result is then passed as a parameter to the components needing it.

P.S: a lot of values are re-reverted when we use the result. Is all this reversing thing necessary ?